core/src/client/async_client/manager.rs

```diff
@@ -32,14 +32,16 @@
 //!    > **Note**: The spec allow number, string or null but this crate only supports numbers.
 //!    - SubscriptionId: unique ID generated by server
 
-use std::collections::{hash_map::Entry, HashMap};
-
 use crate::Error;
+use core::hash::BuildHasherDefault;
 use futures_channel::{mpsc, oneshot};
+use hashbrown::hash_map::{Entry, HashMap};
 use jsonrpsee_types::{Id, SubscriptionId};
-use rustc_hash::FxHashMap;
+use rustc_hash::FxHasher;
 use serde_json::value::Value as JsonValue;
 
+type FxHashMap<K, V> = HashMap<K, V, BuildHasherDefault<FxHasher>>;
```

Comment:
Is `FxHashMap` not wasm-compatible? I would have thought it was defined pretty much like you did here?

Reply:
you are probably right, I don't remember.